crates/connectors/ndc-postgres/src/configuration/version2.rs

```diff
@@ -58,10 +72,19 @@ pub struct ConfigureOptions {
     /// internal schemas of Postgres, Citus, Cockroach, and the PostGIS extension.
     #[serde(default = "default_excluded_schemas")]
     pub excluded_schemas: Vec<String>,
+    /// Deprecated alias for 'unqualifiedSchemasForTables'.
+    #[serde(default)]
+    // weirdly, we have to use 'skip_serializing_if' rather than 'skip_serializing', since the
+    // latter causes schemars to consider the field required, in spite of the 'default' attribute.
+    #[serde(skip_serializing_if = "always_skip_unqualified_schemas")]
+    pub unqualified_schemas: Vec<String>,
```

Comment:
Can you please document the relationship between this field and the others? What happens if this field and the others are defined?